src/scripts/main.js

```diff
@@ -1 +1,9 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === 'menu') {
```

Comment:
Instead of checking for the exact 'menu' hash value, it would be better to check if the hash contains 'menu' to cover cases like '#menu-item'. 💡